### ISSUE TITLE
granulomatous follicular conjunctivitis

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10919]
**Preferred term label:**

**Synonyms**
granulomatous nonsuppurative conjunctivitis

**Definition (free text, please give PubMed ID)**
PMID: 39336466 Parinaud oculoglandular syndrome (POS), first reported by Parinaud in 1889, manifests as acute unilateral **granulomatous conjunctivitis** with ipsilateral lymphadenitis . POS is reportedly caused by cat-scratch disease (CSD), tularemia, sporotrichosis, syphilis, and other diseases. POS is mainly diagnosed based on serum antibody titers and lymph node biopsies. The causative diseases of POS are diverse, and a misdiagnosis may result in a worsening of the condition. Therefore, a reliable diagnosis is of great importance, as it is crucial for the appropriate treatment and management of the condition.
(...) **Follicular conjunctivitis** in the right eye and ipsilateral lymphadenitis led to a clinical diagnosis of POS.

PMID: 39286673 Henri Parinaud, an ophthalmologist from France first described the oculoglandular syndrome in 1889, which has come to be known as the ocular bartonellosis, describing three patients with fever, regional lymphadenopathy (preauricular and submandibular lymph nodes), and a **follicular conjunctivitis**. Parinaud oculoglandular syndrome (POGS) patients usually complain of unilateral eye redness, foreign body sensation, and excessive tearing. Diagnosing POGS is more difficult for physicians as it is considered a rare ophthalmological presentation of cat scratch disease (CSD).

PMID: 32313895 The clinical manifestation of a B. henselae infection highly depends on the immune status of the patient. Immunocompetent patients usually present with typical CSD, which is initially characterized by a papule at the site of inoculation, followed by proximal regional lymphadenopathy that can last weeks to months. Atypical manifestation occurs in 5% of the cases and may include Parinaud’s oculoglandular syndrome (which comprises **granulomatous nonsuppurative conjunctivitis** with adjacent ipsilateral preauricular lymphadenopathy), neuroretinitis, encephalitis, osteomyelitis, endocarditis, hepatosplenic lesions, and fever of unknown origin. In immunocompromised patients CSD may also manifest as bacteremia, bacillary angiomatosis and peliosis.

**Parent term (use hpo.jax.org/app)**
Conjunctivitis HP:0000509

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Cat-scratch disease (CSD) ORPHA:50839

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f this is "bundled", in that there are two phenotypic abnormalities. I am adding two terms, and one could use both to annotate this disease

Adding new term: 
- ID: HP:6001385
- label: Follicular conjunctivitis
- parents: [Conjunctivitis (HP:0000509)](https://hpo.jax.org/app/browse/term/HP:0000509)
- pmid: 
- def: A type of conjunctivitis characterized by the formation of conjunctival follicles, which appear as raised domes on the are round collections of lymphocytes, and are often most prominent in the inferior fornix (interior part of the lower eyelid).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10919
- synonyms:

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6001386
- label: Granulomatous conjunctivitis
- parents: [Conjunctivitis (HP:0000509)](https://hpo.jax.org/app/browse/term/HP:0000509)
- pmid: PMID:37318529
- def: A type of conjunctivitis characterized by the formation of conjunctival granulomata, i.e., localized nodular inflammatory lesions of the conjunctiva, characterized histologically by aggregates of epithelioid histiocytes, often with multinucleated giant cells, surrounded by lymphocytes and fibroblasts.
- comment: 
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10919
- synonyms:
